src/App.tsx

```diff
@@ -1,17 +1,83 @@
-import React from 'react';
-import './App.css';
+/* eslint-disable */ 
+import React, { useState } from 'react';
+import './App.scss';
+import { TodoList } from './components/TodoList/TodoList';
 
 import users from './api/users';
+import todos from './api/todos';
+
+const preparedTodos = todos.map(todo => ({
+  ...todo,
+  user: users.find(user => user.id === todo.userId) || null,
+}));
 
 const App: React.FC = () => {
+  const [newTask, setNewTask] = useState('');
+  const [userFromForm, setUserFromForm] = useState('Leanne Graham');
+  const [allTodos, setAllTodos] = useState(preparedTodos);
+  const [isEmpty, setIsEmpty] = useState(false);
```

Comment:
You don't need this state, you already have state with title so it can be just a variable, `const isEmpty = title.length === 0`